doc/xml/user-guide.xml

```diff
@@ -2417,6 +2456,22 @@
         <p>File creation time in object stores is relatively slow so commands benefit by increasing <br-option>process-max</br-option> to parallelize file creation.</p>
     </section>
 
+    <!-- ======================================================================================================================= -->
+    <section id="sftp-support" if="!{[object-any-all]}" depend="/quickstart/configure-archiving">
+        <title>SFTP Support</title>
+
+        <block id="sftp-setup">
+            <block-variable-replace key="sftp-setup-repo-id">5</block-variable-replace>
+            <block-variable-replace key="sftp-setup-host">{[host-pg1]}</block-variable-replace>
+            <block-variable-replace key="sftp-setup-user">postgres</block-variable-replace>
+            <block-variable-replace key="sftp-setup-config-owner">postgres:postgres</block-variable-replace>
+        </block>
+
+        <p>Commands are run exactly as if the repository were stored on a local disk.</p>
+
```

Comment:
I added stanza-create here:
```
        <execute-list host="{[host-pg1]}">
            <title>Create the stanza</title>

            <execute user="postgres" output="y">
                <exe-cmd>{[project-exe]} {[dash]}-stanza={[postgres-cluster-demo]} {[dash]}-log-level-console=info stanza-create</exe-cmd>
                <exe-highlight>completed successfully</exe-highlight>
            </execute>
        </execute-list>
```
And got this error:
```
P00  ERROR: [049]: timeout connecting to 'repository:22'
```
We should probably do a backup here as well to make sure things are really working.

Reply:
If I'm looking correctly, the doc build hosts are u18 using libssh2 v1.8 which requires a PEM format rsa keypair. If it won't break other things, we can add -m PEM to the current keygen command.  

When I add the stanza-create into the sftp-support section I get an error re missing a gcs file prior to attempting the stanza-create for the sftp repo. Pointers to get around that?

```2023-05-03 18:14:48.253 P00   INFO:         process section: /sftp-support
2023-05-03 18:14:49.448 P00  ERROR: [125]: command 'docker exec -u vagrant doc-pg-primary bash -l -c 'sudo -u postgres pgbackrest --stanza=demo --log-level-console=info stanza-create'' returned 55
                                        STDOUT (last 10,000 characters):
                                        P00   INFO: stanza-create command begin 2.46dev: --exec-id=2924-74aaf0c9 --log-level-console=info --log-level-stderr=off --no-log-timestamp --pg1-path=/var/lib/postgresql/12/demo --repo2-azure-account=<redacted> --repo2-azure-container=demo-container --repo2-azure-key=<redacted> --repo1-cipher-pass=<redacted> --repo1-cipher-type=aes-256-cbc --repo4-gcs-bucket=demo-bucket --repo4-gcs-key=<redacted> --repo1-path=/var/lib/pgbackrest --repo2-path=/demo-repo --repo3-path=/demo-repo --repo4-path=/demo-repo --repo5-path=/demo-repo --repo3-s3-bucket=demo-bucket --repo3-s3-endpoint=s3.us-east-1.amazonaws.com --repo3-s3-key=<redacted> --repo3-s3-key-secret=<redacted> --repo3-s3-region=us-east-1 --repo5-sftp-account=pgbackrest --repo5-sftp-host=repository --repo5-sftp-hostkey-hash-type=sha1 --repo5-sftp-private-keyfile=/home/pgbackrest/.ssh/id_rsa --repo5-sftp-public-keyfile=/home/pgbackrest/.ssh/id_rsa.pub --repo2-type=azure --repo3-type=s3 --repo4-type=gcs --repo5-type=sftp --stanza=demo
                                        P00   INFO: stanza-create for stanza 'demo' on repo1
                                        P00   INFO: stanza 'demo' already exists on repo1 and is valid
                                        P00   INFO: stanza-create for stanza 'demo' on repo2
                                        P00   INFO: stanza 'demo' already exists on repo2 and is valid
                                        P00   INFO: stanza-create for stanza 'demo' on repo3
                                        P00   INFO: stanza 'demo' already exists on repo3 and is valid
                                        P00   INFO: stanza-create for stanza 'demo' on repo4
                                        P00  ERROR: [055]: unable to open missing file '/etc/pgbackrest/gcs-key.json' for read
                                        P00   INFO: stanza-create command end: aborted with exception [055]```

Reply:
Can we fix this by just updating to Ubuntu 20.04? I tried and that didn't seem to work. I guess I'm fine with updating key-gen, but I don't know if it will break anything else.

As for GCS, looks like we'll need to swap the sections. I have done that in 916d5b40d44 (and updated the OS to 20.04). Why don't you go ahead and make the SSH changes required?